pkg/cmd/util/cmd_test.go

```diff
@@ -1,229 +0,0 @@
-package util_test
```

Comment:
Even if we use discover here (iirc) we should still test it's working as expected. Those few tests should not hurt and yet might save us from breaking stuff. Unless there's different tests verifying thse.